packages/client/src/scripts/check-word-mute.ts

```diff
@@ -1,23 +1,27 @@
-export function checkWordMute(note: Record<string, any>, me: Record<string, any> | null | undefined, mutedWords: string[][]): boolean {
+export function checkWordMute(note: Record<string, any>, me: Record<string, any> | null | undefined, mutedWords: Array<string | string[]>): boolean {
 	// 自分自身
 	if (me && (note.userId === me.id)) return false;
 
-	const words = mutedWords
-		// Clean up
-		.map(xs => xs.filter(x => x !== ''))
-		.filter(xs => xs.length > 0);
-
-	if (words.length > 0) {
+	if (mutedWords.length > 0) {
 		if (note.text == null) return false;
 
-		const matched = words.some(and =>
-			and.every(keyword => {
-				const regexp = keyword.match(/^\/(.+)\/(.*)$/);
-				if (regexp) {
+		const matched = mutedWords.some(filter => {
+			if (Array.isArray(filter)) {
+				return filter.every(keyword => note.text!.includes(keyword))
+			} else {
+				// represents RegExp
+				const regexp = line.match(/^\/(.+)\/(.*)$/);
+				// This should never happen due to input sanitisation.
+				if(!regexp)
+					return false;
+				try {
 					return new RegExp(regexp[1], regexp[2]).test(note.text!);
+				} catch (err: SyntaxError) {
```

Comment:
You can't write type annotation in catch clause (TS1196).

```suggestion
				} catch (err) {
```

Reply:
Ah I see. I thought I read I could do that, but apparently I did not read thoroughly enough that i can only use `any` or `unknown`. I assume the same applies in `packages/backend/src/misc/check-word-mute.ts`?

Reply:
yes.